### PR TITLE
config: enable contribution for docs sig

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

Enable contribution plugin, which will add `contribution` or `first-time-contributor` labels to PR accordingly.

### CODE REVIEW

**Comment by ran-huang** [comment]
@yikeke PTAL.
I just found that ti-sre-bot hasn't added any `first-time-contributor` labels to PRs for a few weeks. We need to enable this tichi plugin.

File: prow/config/external_plugins_config.yaml [hunk 1]
```diff
@@ -930,6 +930,10 @@ ti-community-contribution:
       - pingcap/ticdc
       - pingcap/br
       - chaos-mesh/chaos-mesh
+      - pingcap/docs
```
Comment (hi-rustin) [inline prow/config/external_plugins_config.yaml:933]: Need to add the corresponding event config, see: https://github.com/ti-community-infra/configs/blob/3bd74980b1dc6a11525d559dacc3515400311037/prow/config/plugins.yaml#L336

**Review by hi-rustin — CHANGES_REQUESTED** [review]

**Review by yikeke — APPROVED** [review]

**Review by hi-rustin — APPROVED** [review]

**Pull request merged**